pkg/scheduler/framework/plugins/noderesources/test_util.go

```diff
@@ -29,19 +29,7 @@ var (
 )
 
 func makeNode(node string, milliCPU, memory int64) *v1.Node {
-	return &v1.Node{
-		ObjectMeta: metav1.ObjectMeta{Name: node},
-		Status: v1.NodeStatus{
-			Capacity: v1.ResourceList{
-				v1.ResourceCPU:    *resource.NewMilliQuantity(milliCPU, resource.DecimalSI),
-				v1.ResourceMemory: *resource.NewQuantity(memory, resource.BinarySI),
-			},
-			Allocatable: v1.ResourceList{
-				v1.ResourceCPU:    *resource.NewMilliQuantity(milliCPU, resource.DecimalSI),
-				v1.ResourceMemory: *resource.NewQuantity(memory, resource.BinarySI),
-			},
-		},
-	}
+	return makeNodeWithExtendedResource(node, milliCPU, memory, nil)
```

Comment:
since `makeNode` is an internal function to this package, could you just update each call to `makeNode` to use `makeNodeWithExtendedResource`? It doesn't seem like there is a reason to wrap this function since we don't have to worry about deprecation and other consumers